domino_data/_feature_store/client.py

```diff
@@ -44,17 +44,30 @@ def __attrs_post_init__(self):
             ),
         )
 
-    def post_feature_views(self, feature_views: List[FeatureViewRequest], commit_id: str) -> None:
+    def post_feature_views(
+        self,
+        feature_views: List[FeatureViewRequest],
+        commit_id: str,
+        project_id: str,
+        user_id: str,
+    ) -> None:
         """Insert or update feature views.
 
         Args:
             feature_views: an array of feature views to be inserted or updated.
             commit_id: the commit hash the feature store is to be synced with
+            project_id: the id of the project from where the sync operation is performed
+            user_id: the id of the user who starts this sync operation
 
         Raises:
             ServerException: if update fails
         """
-        request = UpsertFeatureViewsRequest(feature_views=feature_views, git_commit_hash=commit_id)
+        request = UpsertFeatureViewsRequest(
+            feature_views=feature_views,
+            git_commit_hash=commit_id,
+            project_id=project_id,
+            user_id=user_id,
```

Comment:
should this come from the principal? or are we ok with it being passed by the client